app/models/badges/researcher_badge.rb

```diff
@@ -0,0 +1,40 @@
+module Badges
+  class ResearcherBadge < Badge
+    seed "Researcher",
+      :ultimate,
+      'researcher',
+      'Awarded for having helped with Exercism Research'
+
+    def award_to?(user)
+      RESEARCHERS.include?(user.handle.downcase)
```

Comment:
Are the handles exercism handles or GH handles?

Reply:
Let me double check

Reply:
They're github usernames.